agent/Makefile

```diff
@@ -40,7 +40,6 @@ util-scripts = \
 	pbench-init-tools \
 	pbench-kill-tools \
 	pbench-list-tools \
-	pbench-list-triggers \
```

Comment:
While we want to remove it from here, we need to add it below just like `pbench-config`.